stores-api/assets-api/src/main/java/com/bloxbean/cardano/yaci/store/api/assets/controller/AssetController.java

```diff
@@ -2,96 +2,129 @@
     }
 
-    @GetMapping("/assets/txs/unit/{unit}")
-    public List<TxAsset> getAssetTxsByUnit(@PathVariable String unit, @RequestParam(name = "page", defaultValue = "0") int page,
-                                         @RequestParam(name = "count", defaultValue = "10") int count) {
+    @Tag(name = "Asset Service")
+    @GetMapping("/policy/{policyId}")
+    @Operation(summary = "Policy Assets", description = "Returns the information for all assets under the same policy.")
+    public List<TxAsset> getAssetTxsByPolicyId(@PathVariable String policyId, @RequestParam(name = "page", defaultValue = "0") @Min(0) int page,
+                                             @RequestParam(name = "count", defaultValue = "10") @Min(1) @Max(100) int count) {
         //TODO -- Fix pagination index
         int p = page;
         if (p > 0)
             p = p - 1;
 
-        if (count > 100)
-            throw new IllegalArgumentException("Max no of records allowed is 100");
-
-        return assetService.getAssetTxsByUnit(unit, page, count);
+        return assetService.getAssetTxsByPolicyId(policyId, page, count);
     }
 
-    @GetMapping("/assets/supply/fingerprint/{fingerprint}")
+    @Tag(name = "Asset Service")
+    @GetMapping("/supply/fingerprint/{fingerprint}")
+    @Operation(summary = "Assets Supply by Fingerprint", description = "Returns the entire supply of a specific asset by fingerprint.")
     public FingerprintSupply getSupplyByFingerprint(@PathVariable String fingerprint) {
         return assetService.getSupplyByFingerprint(fingerprint)
                 .map(supply -> new FingerprintSupply(fingerprint, supply))
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Asset not found"));
     }
 
-    @GetMapping("/assets/supply/unit/{unit}")
+    @Tag(name = "Asset Service")
+    @GetMapping("/supply/unit/{unit}")
+    @Operation(summary = "Assets Supply by Unit", description = "Returns the entire supply of a specific asset by unit.")
     public UnitSupply getSupplyByUnit(@PathVariable String unit) {
         return assetService.getSupplyByUnit(unit)
                 .map(supply -> new UnitSupply(unit, supply))
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Asset not found"));
     }
 
-    @GetMapping("/assets/supply/policy/{policy}")
+    @Tag(name = "Asset Service")
+    @GetMapping("/supply/policy/{policy}")
+    @Operation(summary = "Assets Supply by Policy", description = "Returns the entire assets supply of a specific policy.")
     public PolicySupply getSupplyByPolicy(@PathVariable String policy) {
         return assetService.getSupplyByPolicy(policy)
                 .map(supply -> new PolicySupply(policy, supply))
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Asset not found"));
     }
 
 
-    record FingerprintSupply(String fingerprint, int supply) {
```

Comment:
I changed it because of this intellij Inspection:
![image](https://github.com/bloxbean/yaci-store/assets/3818746/39249dc7-f27d-42e6-9e0d-e2f628a4b0d4)